src/Core/CommandBus/Parser/CommandHandlerDefinition.php

```diff
@@ -135,6 +135,14 @@ public function getCommandClass(): string
         return $this->commandClass;
     }
 
+    /**
+     * @return string
+     */
+    public function getSimpleCommandClass(): string
+    {
+        return substr($this->commandClass, strrpos($this->commandClass, '\\') + 1);
```

Comment:
```suggestion
        return substr($this->getCommandClass(), strrpos($this->getCommandClass(), '\\') + 1);
```

Always prefer accessor, otherwise put the class in final if it can't be extend